cli/src/cmd_auth.rs

```diff
@@ -699,107 +594,107 @@ mod tests {
             Ok(host) if host == "http://example.com:8888/"
         ));
     }
-}
 
-#[test]
-fn test_cmd_auth_status() {
-    use assert_cmd::Command;
-    use httpmock::{Method::GET, MockServer};
-    use predicates::str;
-
-    let server = MockServer::start();
-    let oxide_mock = server.mock(|when, then| {
-        when.method(GET)
-            .path("/v1/me")
-            .header("authorization", "Bearer oxide-token-1111");
-        then.status(200).json_body_obj(&oxide::types::CurrentUser {
-            display_name: "privileged".to_string(),
-            id: "001de000-05e4-4000-8000-000000004007".parse().unwrap(),
-            silo_id: "d1bb398f-872c-438c-a4c6-2211e2042526".parse().unwrap(),
-            silo_name: "funky-town".parse().unwrap(),
+    #[test]
+    fn test_cmd_auth_status() {
+        use assert_cmd::Command;
+        use httpmock::{Method::GET, MockServer};
+        use predicates::str;
+
+        let server = MockServer::start();
+        let oxide_mock = server.mock(|when, then| {
+            when.method(GET)
+                .path("/v1/me")
+                .header("authorization", "Bearer oxide-token-1111");
+            then.status(200).json_body_obj(&oxide::types::CurrentUser {
+                display_name: "privileged".to_string(),
+                id: "001de000-05e4-4000-8000-000000004007".parse().unwrap(),
+                silo_id: "d1bb398f-872c-438c-a4c6-2211e2042526".parse().unwrap(),
+                silo_name: "funky-town".parse().unwrap(),
+            });
```

Comment:
let's use the oxide-httpmock typed interface

Reply:
Removed with https://github.com/oxidecomputer/oxide.rs/pull/727